docs/source/admin/traffic_ops/using.rst

```diff
@@ -881,7 +892,9 @@ The CZF is an input to the Traffic Control CDN, and as such does not get generat
 
 The script that generates the CZF file is not part of Traffic Control, since it is different for each situation.
 
-.. note:: The ``"coordinates"`` section is optional and may be used by Traffic Router for localization in the case of a CZF "hit" where the zone name does not map to a Cache Group name in Traffic Ops (i.e. Traffic Router will route to the closest Cache Group(s) geographically).
+.. note:: The ``"coordinates"`` section is optional and may be used by Traffic Router for localization in the case of a CZF "hit" where the zone name does not map to a Cache Group name in Traffic Ops (i.e. Traffic Router will route to the closest Cache Group(s) geographically).  
+
+The ``"backupList"`` section is optional and may be used by Traffic Router for localization in the case of a CZF "hit" where the zone name does not map to a Cache Group name in Traffic Ops (i.e. Traffic Router will route to the configured backup Cache Group(s)). This can be used as an alternative to ``"coordinates"`` therby enabling operators to have a control over choosing back up edge servers rather than depending on Geo-coordinates based routing.
```

Comment:
Please also mention that the list is ordered. 
Also, include that if there are no caches available in any of the listed groups, the request will be bypassed (if configured) or rejected